gcc/config.gcc

```diff
@@ -1264,24 +1264,40 @@ aarch64*-*-linux*)
 	TM_MULTILIB_CONFIG=`echo $TM_MULTILIB_CONFIG | sed 's/^,//'`
 	;;
 aarch64*-*-mingw*)
-# 	tm_file="${tm_file} dbxelf.h elfos.h gnu-user.h linux.h glibc-stdint.h"
-# 	tm_file="${tm_file} aarch64/aarch64-elf.h aarch64/aarch64-errata.h aarch64/aarch64-linux.h"
-# 	tm_file="${tm_file} i386/unix.h i386/bsd.h i386/gas.h dbxcoff.h i386/cygming.h"	
-	tm_file="${tm_file} aarch64/aarch64-coff.h"
-	tm_file="${tm_file} aarch64/cygming.h"
+	tm_file="${tm_file} aarch64/aarch64-coff.h aarch64/cygming.h" # i386/unix.h i386/bsd.h i386/gas.h
+	xm_file=i386/xm-mingw32.h
```

Comment:
is it really needed?

Reply:
Not sure about that but the content of the file seems as something generic accross mingw targets. As a followup of the stage 1 patch set merge, we can refactor and move those files to the shared `mingw` configuration folder. I didn't want to do such big refactor now.

Reply:
I would add only changes which are needed for enabled-shared fix

Reply:
Ok, then I'll move them to https://github.com/Windows-on-ARM-Experiments/gcc-woarm64/pull/14 and retarget that branch. I'll keep this PR open as WIP for future reference.